kiliautoml/models/_hugging_face_named_entity_recognition_model.py

```diff
@@ -249,6 +165,73 @@ def tokenize_and_align_labels(examples):
         trainer.save_model(ensure_dir(path_model))
         return output.training_loss
 
+    def predict(
+        self,
+        *,
+        assets: List[AssetT],
+        model_path: Optional[str],
+        from_project: Optional[str],
+        batch_size: int,
+        verbose: int,
+        clear_dataset_cache: bool,
+    ) -> JobPredictions:
+        _ = clear_dataset_cache
+        print("Warning, this method does not support custom batch_size")
```

Comment:
`kili_print`

Reply:
or `warn`. But relates to the previous comment.

Reply:
Maybe we should create a ticket and start using the logging module

Reply:
with flake8-print-clean precommit